test/unit/alignment/configuration/align_config_edit_test.cpp

```diff
@@ -30,7 +32,7 @@ TEST(align_cfg_edit, is_hamming)
 
 TEST(align_cfg_edit, is_simple_gap)
 {
-    auto scheme = seqan3::get<seqan3::align_cfg::gap>(seqan3::align_cfg::edit_scheme).value;
-    EXPECT_EQ(scheme.get_gap_score(), -1);
-    EXPECT_EQ(scheme.get_gap_open_score(), 0);
+    auto gap_cost_scheme = get<seqan3::align_cfg::gap_cost_affine>(seqan3::align_cfg::edit_scheme);
```

Comment:
```suggestion
    auto gap_cost_scheme = seqan3::get<seqan3::align_cfg::gap_cost_affine>(seqan3::align_cfg::edit_scheme);
```

Is this not working?

Reply:
no

Reply:
Thank you :)